data/bash-completion/fwupdmgr

```diff
@@ -13,6 +13,8 @@ _fwupdmgr_cmd_list=(
 	'emulation-untag'
 	'emulation-load'
 	'emulation-save'
+	'fix-host-security-attr'
+	'unfix-host-security-attr'
```

Comment:
Since this has to do with `security` command specifically How about if we rename this to:
```
security-fix-attr
security-unfix-attr
```

Reply:
`security-fix` makes sense indeed; in the future we might want to prompt for the ID rather than expecting it. Fixed.